src/app/prescription/medication-request-form/medication-form/medication-form.component.css

```diff
@@ -24,3 +24,16 @@ button.mat-stroked-button {
   font-weight: 400;
   line-height: 1.125;
 }
+
+.container-ingredient-amount-form {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 500px;
+}
+
+.ingredient-field,
```

Comment:
Same here, maybe give it a more generix name ( "short-field", "medium-field", "large-field" with 30, 50, 100% or something like that ?)